slam-ws/src/caffeine/urdf/caffeine.urdf.xacro

```diff
@@ -306,6 +306,49 @@
       <origin xyz="${base_width / 2.0} 0 ${base_height / 2.0 + hokuyo_height / 2.0 + 0.5}" rpy="0 0 ${-135 * pi / 180}"/>
    </joint>
 
+   <!-- ZED Camera -->
+  <link name="zed_camera_link">
+      <visual>
+         <origin xyz="0 0 0" rpy="0 0 0"/>
+            <!-- TODO: should be using the mesh but couldn't get it to work -->
+         <geometry>
+            <box size="${ZED_camera_length} ${ZED_camera_width} ${ZED_camera_height}"/>
+         </geometry>
+       </visual>  
+       <collision>
+           <origin xyz="0 0 0" rpy="0 0 0"/>
+           <geometry>
+               <box size="${ZED_camera_length} ${ZED_camera_width} ${ZED_camera_height}"/>
+           </geometry>
+       </collision>
+       <inertial>
+           <mass value="${ZED_camera_mass}"/> <!-- Obtained from google search -->
+           <xacro:box_inertia mass="${ZED_camera_mass}" length="${ZED_camera_length}" width="${ZED_camera_width}" height="${ZED_camera_height}"/>
+       </inertial>
+   </link>
+   <joint name="zed_camera_joint" type="fixed">
+       <parent link="base_link"/>
+       <child link="zed_camera_link"/>
+       <origin xyz="${stand_length/2} 0 ${stand_height + ZED_camera_height}" rpy="${pi/4} 0 ${pi/2}"/>
```

Comment:
add  `+ base_height/2` to `${stand_height + ZED_camera_height}` since the base plate has a thickness

Reply:
Done